arm/examples/create/create.go

```diff
@@ -1,16 +1,21 @@
-package create
+package main
 
 import (
 	"fmt"
 	"log"
+	"math/rand"
+	"time"
 
 	"github.com/Azure/azure-sdk-for-go/arm/examples/helpers"
 	"github.com/Azure/azure-sdk-for-go/arm/storage"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-func createAccount(resourceGroup, name string) {
+func main() {
+	resourceGroup := "resourceGroupName"
+	name := fmt.Sprintf("gosdktestname%s", randomString(5))
+
 	c, err := helpers.LoadCredentials()
```

Comment:
I think ideally at some point we should get rid of this helper too. It is reading a non-standard credentials file format just to construct an in-memory object. We can just hardcode stuff like 

```
clientID = "replace-this"
clientSecret = "replace-this"
tenantID = "replace-this"
subsID = "replace-this"

resourceGroupName="replace-this"
```

and it'll be clear to anyone that how we actually construct the service principal token object inline.

cc: @colemickens